bases/br_sp_seduc_inse/dataset_config.yaml

```diff
@@ -0,0 +1,76 @@
+dataset_id: br_sp_seduc_inse # AUTO GENERATED
+
+url_ckan: https://basedosdados.org/dataset/br-sp-seduc-inse
+url_github: https://github.com/basedosdados/mais/tree/master/bases/br_sp_seduc_inse
+
+# Descreva a base.
+# Ela é sobre o que?
+# Quais as principais fontes de dados?
+# Há links para FAQs e explicações?
+description: | # REQUIRED
+    A base contém a tabela sobre o Indicador Socioeconomico das Escolas do Estado de São Paulo, o INSE. O indicador varia de 0 a 10, sendo 10 a escola com o nível socioeconômico mais baixo e 0 a escola com nível mais alto. O INSE é calculado a partir dos questionários do SARESP. Ele considera as seguintes variáveis: grau de escolaridade dos pais; posse de bens de consumo duráveis na residência; renda familiar.  varia de 0 a 10, sendo 10 a escola com o nível socioeconômico mais baixo e 0 a escola com nível mais alto.
+
```

Comment:
Socioeconômico.

Excluir "varia de 0 a 10, sendo 10 a escola com o nível socioeconômico mais baixo e 0 a escola com nível mais alto" do início ou do final da descrição.